Week2/solution-2.txt

```diff
@@ -0,0 +1,44 @@
+
+1. Create a table of student and student details.
+Ans:-	
+	create table stud21
+	(
+		id text primary key,
+		name text
+	);
+	
+	create table stud_detail21
+	(
+		id text not null,
+		city text,	
+		m_no number,
+		foreign key(id) references stud16(id)
+	);
+
+2. Insert minimum 5 records in each table.
+Ans:- 
+	insert into stud21
+	values('1','Arpi'),('2','Astha'),('3','RAM'),('4','OM'),('5','sai');
+	
+	insert into stud_detail21
+	values('1','SURAT',67482374456),('2','BARDOLI',9753443210),('3','VAPI',9867453546),('4','SURAT',8764563456),('5','SURAT',8762346543);
+
+3. Display student details who come from the surat city.
+Ans:-
+	select s.id,s.name,sd.city,sd.m_no
+   	from stud21 s join stud_detail16 sd
+    	on s.id=sd.id
+    	where city='surat';
+

```

Comment:
Display student details who come from the Bardoli and display student name started with 'R'.